packages/webpack-plugin/src/loader.ts

```diff
@@ -61,7 +83,8 @@ export default kumaUiLoader;
 const requireReact = (code: string, id: string) => {
   if (id.endsWith(".jsx") || id.endsWith(".tsx")) {
     if (!/^\s*import\s+React\s+from\s+['"]react['"]/.test(code)) {
-      return "import React from 'react';\n" + code;
+      // return "import React from 'react';\n" + code;
+      return code;
```

Comment:
Could you explain the reason for commenting out this line and changing it to `return code;`? I may be misremembering, but I thought we had to import React because we're using 'classic' JSX runtime 👀

Reply:
The reason for removing this is that the 'use clinet'; needs to be on the first line, and adding it would cause a crash. Also, I don't think adding 'react import' is a style library responsibility.